README.md

```diff
@@ -1 +1,52 @@
-# re-observe-test-server
\ No newline at end of file
+# Prometheus scrape endpoint generator
+
+Application to be run as part of a Prometheus load test which exposes multiple target endpoints for Prometheus to scrape.
+
+Endpoints are provided at `/metrics/<endpoint_number>` for example `/metrics/2`. The metrics returned include a range of
+counters, gauges and histograms with random values.
+
+This tool is designed to be used in conjunction with the load test tool and the [Prometheus terraform repo](https://github.com/alphagov/prometheus-aws-configuration-beta).
+
+It is based off the [Prometheus prombench fake-webserver](https://github.com/prometheus/prombench/tree/master/apps/fake-webserver) and has been adapted to make it easier to generate the necessary metrics and size of time series needed for load testing at a number of different loads.
```

Comment:
the link to prombench fake-webserver 404s

Reply:
Had been moved, I've updated it with a new link